packages/castable-video/castable-remote-playback.js

```diff
@@ -20,7 +20,12 @@ const castElementRef = new WeakSet();
 let cf;
 
 onCastApiAvailable((isAvailable) => {
-  if (isAvailable && !cf) {
+  if (!isAvailable) {
+    console.warn('Cast API is not available.');
```

Comment:
question(non-blocking): Do we actually want to warn in this case? If this might be legitimately invoked for cases where it's valid to use the castable video mixin (e.g. a web component on another browser, cases where chrome version may not have casting supported, etc.), we probably should not log a warning (or at least that's my S1).